packages/core/core/src/AssetGraph.js

```diff
@@ -542,7 +542,7 @@ export default class AssetGraph extends ContentGraph<AssetGraphNode> {
 
   getIncomingDependencies(asset: Asset): Array<Dependency> {
     let nodeId = this._contentKeyToNodeId.get(asset.id);
-    if (!nodeId) {
+    if (nodeId == null) {
```

Comment:
Shouldn't this throw if we're asking for an asset that the graph doesn't know about?

Reply:
Doesn't seem like it — tests continue to pass. Updated it to do this 😄